dense-linear-algebra/dense-linear-algebra.cabal

```
@@ -0,0 +1,50 @@
+name:                dense-linear-algebra
+version:             0.1.0.0
```

```suggestion
version:             0.2.0.0
```

This should be higher than the latest Hackage version I think

---

I usually bump right before release since you don't always know how much do you need to bump